ethers-middleware/tests/nonce_manager.rs

```diff
@@ -44,12 +44,12 @@ async fn nonce_manager() {
     }
 
     // sleep a bit to ensure there's no flakiness in the test
-    std::thread::sleep(std::time::Duration::new(5, 0));
+    tokio::time::sleep(Duration::from_secs(10)).await;
```

Comment:
why are we changing these from thread sleeps to tokio ones and making them longer?

Reply:
std sleep blocks the entire tokio runtime or something like that, longer to hopefully make the provider find the txs since it keeps failing in ci